modules/running-cluster-loader.adoc

```diff
@@ -5,14 +5,19 @@
 [id="running_cluster_loader_{context}"]
 = Running Cluster Loader
 
+.Prerequisites
+
+* The repository needs authentication. Use your existing authentication
```

Comment:
My only question would be, what does it mean for a repository to need authentication?